docs/Routes.md

```diff
@@ -433,3 +434,55 @@ If the request will not have the `Accept-Version` header, a 404 error will be re
 
 #### Custom
 It's possible to define a custom versioning logic. This can be done through the [`versioning`](https://github.com/fastify/fastify/blob/master/docs/Server.md#versioning) configuration, when creating a fastify server instance.
+
+<a name="analyzing-routes"></a>
+### Analyzing Routes
+
+#### Getting Routes
+
+An array of registered routes can be dumped with the `fastify.registeredRoutes` method:
+
+```js
+fastify.get('/hello', helloHandler)
+fasttify.ready(() => {
+  console.log(fastify.registeredRoutes())
+})
+```
+
+The method returns an array of objects with keys: `method`, `path`, `opts`, `handler` and `store`. 
+
+The above would print an array similar to the following:
+
+```js
+[{  method: 'GET',
+    path: '/hello',
+    opts: { version: undefined },
+    handler: [Function: routeHandler],
+    store: Context {...} 
+}]
+```
+
+#### Printing Routes
+
+A ASCII tree map of routes can be generated with the `fastify.printRoutes` method:
```

Comment:
```suggestion
A Radix tree of routes can be generated with the [`fastify.printRoutes`](/Server.md#printroutes) method:
```

Reply:
@Eomm wouldn't "radix tree" be the conceptual definition of the implementation? What is actually printed is an ASCII representation of it.

Reply:
What I would like to say to the user is that the output could be not what he expects for this old issue https://github.com/fastify/fastify/issues/1581